src/components/PullRequestList/index.tsx

```diff
@@ -1,11 +1,11 @@
-import { FC, useState, useEffect } from 'react';
+import React, { FC, useState, useEffect } from 'react';
```

Comment:
Next.js doesn't need `React` import I guess, Please confirm and do the needful.

Reply:
yeah we dont need it.